tex/set-theory/forcing.tex

```diff
@@ -447,7 +493,7 @@ \section{Fundamental theorem of forcing}
 \begin{quote}
 	For \emph{any} generic $G$,
 	if $p \in G$,
-	then $M[G] \vDash \varphi(\sigma_1^G, \dots, \sigma_n^G)$.
+	then $M[G] \vDash \varphi[\sigma_1^G, \dots, \sigma_n^G]$.
```

Comment:
I think I added a box before explaining the reasoning behind using φ(...) versus φ[...] notation (it's not that clear I think)